components/BottomNavigation/src/MDCBottomNavigationBarController.m

```diff
@@ -325,33 +351,12 @@ - (void)handleNavigationBarLongPressEndedForPoint:(CGPoint)point {
 
 #pragma mark - Private Methods
 
-/**
- * Removes the given view controller from its parent view controller and its view from its superview
- */
-- (void)removeContentViewController:(UIViewController *)viewController {
-  [viewController removeFromParentViewController];
-  [viewController.view removeFromSuperview];
-}
-
-/**
- * Adds the given view controller to the view controller hierarchy and its view to the content
- * view.
- */
-- (void)addContentViewController:(UIViewController *)viewController {
-  BOOL doesNotContainViewController = ![self.childViewControllers containsObject:viewController];
-  if (viewController && doesNotContainViewController) {
-    [self addChildViewController:viewController];
-    [self.content addSubview:viewController.view];
-    [self addConstraintsForChildViewControllerView:viewController.view];
-    [viewController didMoveToParentViewController:self];
+- (void)removeExistingViewControllers {
+  NSArray<UIViewController *> *childViewControllers = self.childViewControllers;
+  for (UIViewController *childViewController in childViewControllers) {
+    [childViewController.view removeFromSuperview];
```

Comment:
I think we also need to call `[childViewController willMoveToParentViewController:nil];` before we remove the view controller as a child and remove its view from the hierarchy.

Reply:
Sorry didn't see the previous review.